doc/devdocs/modules/launcher/plugins/timeZone.md

```diff
@@ -0,0 +1,150 @@
+| `Offset`            | **No**   | String            |
+| `Name`              | Yes      | String            |
+| `MilitaryName`      | Yes      | String            |
+| `Shortcut`          | Yes      | String            |
+| `TimeNamesStandard` | Yes      | List with strings |
+| `TimeNamesDaylight` | Yes      | List with strings |
+| `ShortcutsStandard` | Yes      | List with strings |
+| `ShortcutsDaylight` | Yes      | List with strings |
+| `CountriesStandard` | Yes      | List with strings |
+| `CountriesDaylight` | Yes      | List with strings |
+
+A minimum entry for the `TimeZone.json` looks like:
+
+```json
+  {
+    "Offset": "11:55",
+    "Name": "My crazy time zone",
+  }
+```
+
+A full entry for the `TimeZone.json` looks like:
+
+```json
+  {
+    "Offset": "11:55",
+    "Name": "My crazy time zone",
+    "Shortcut" : "MYTZ",
+    "MilitaryName" : "Order Time Zone",
+    "TimeNamesStandard": [
+        "My crazy standard time"
+    ],
+    "ShortcutsStandard": [
+        "MCST"
+    ],
+    "TimeNamesDaylight": [
+        "My crazy daylight time"
+    ],
+    "ShortcutsDaylight": [
+        "MCDT"
+    ],
+    "CountriesStandard": [
+      "Crazy Land East"
+    ],
+    "CountriesDaylight": [
+      "Crazy Land West"
+    ]
+  }
+```
+
+### Remarks
+
+* At minimum one of the optional value should be filled.
+
+## Scores
+
+* Scores are not used
+
+## Important for developers
+
+### General
+
+* The assembly name is cached into `_assemblyName` (to avoid to many calls of `Assembly.GetExecutingAssembly()`)
+
+## Microsoft.PowerToys.Run.Plugin.TimeZone project
+
+### Important plugin values (meta-data)
+
+| Name            | Value                                                |
+| --------------- | ---------------------------------------------------- |
+| ActionKeyword   | `&`                                                  |
+| ExecuteFileName | `Microsoft.PowerToys.Run.Plugin.TimeZone.dll`        |
+| ID              | `BADD1B06EF0A4B61AD95395F24241D69`                   |
+
+### Interfaces used by this plugin
+
+The plugin use only these interfaces (all inside the `Main.cs`):
+
+* `Wox.Plugin.IPlugin`
+* `Wox.Plugin.IContextMenu`
+* `Wox.Plugin.IPluginI18n`
+* `Wox.Plugin.ISettingProvider`
+* `IDisposable`
+
+### Program files
+
+| File                                  | Content                                                                 |
+| ------------------------------------- | ----------------------------------------------------------------------- |
+| `Classes\OneTimeZone.cs`              | A class that represent one time zone                                    |
+| `Classes\TimeZones.cs`                | A wrapper class that only contains a list with time zones  (see 1)      |
+| `Classes\TimeZoneSettings.cs`         | A class that contains all settings for the Time Zone plugin             |
+| `Extensions\SringBuilderExtension.cs` | Extension methods for `StringBuilder` Objects                           |
```

Comment:
This might require growing the table...

```suggestion
| `Extensions\StringBuilderExtension.cs` | Extension methods for `StringBuilder` Objects                           |
```